homeassistant/components/sensor/tellduslive.py

```diff
@@ -85,7 +85,9 @@ def name(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        if self._type == SENSOR_TYPE_TEMP:
+        if not self.available:
+            return STATE_UNKNOWN
```

Comment:
@molobrakos is right, `state` property should not be queried when `available` returns false. That should be fixed here: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity.py#L208-L227

@Danielhiversen is also right. We should return `None` and let the core handle how to represent if there is no state.